### ISSUE TITLE
Errors in chromium

### ISSUE THREAD

**Issue by elroncio** [voidlinux/void-packages#4682]
After updating to chromium-52.0.2743.116_1 chromium don't work properly.
All the extensions crash and I can't open any website.

From CLI I get this:
`1351:1351:0828/201623:ERROR:platform_thread_posix.cc(118)] pthread_create: Argomento non valido
[1351:1351:0828/201623:ERROR:chrome_browser_main_posix.cc(290)] Failed to create shutdown detector task.
Received signal 4 ILL_ILLOPN 5585dde88360
`

**Comment by chneukirchen** [comment]
Seems to be a seccomp issue, works here with `--disable-seccomp-filter-sandbox`.

**Comment by chneukirchen** [comment]
Likely culprit:

```
open("/proc/self/exe", O_RDONLY)        = -1 ENOENT (No such file or directory)
sendmsg(3, {msg_name=NULL, msg_namelen=0, msg_iov=[{iov_base="NACLHELPER_OK\0", iov_len=14}], msg_iovlen=1, msg_controllen=0, msg_flags=0}, MSG_NOSIGNAL) = 14
recvmsg(3, {msg_name=NULL, msg_namelen=0, msg_iov=[{iov_base="", iov_len=2048}], msg_iovlen=1, msg_controllen=0, msg_flags=0}, 0) = 0
+++ killed by SIGKILL +++
```

Via arch forum: https://bugzilla.redhat.com/show_bug.cgi?id=1361157

**Comment by xtraeme** [comment]
Fixed by @Duncaen with 5b24a5047b3e374bc0138ea3401f35069cb22cd5
